config/networks/ganache/chainlink.js

```diff
@@ -0,0 +1,6 @@
+
+module.exports = {
+    dataContracts: {
+        eth_usd: '0x1234567890123456789012345678901234567890',
```

Comment:
What is the use of this fake address right now? Should we not be putting in the correct contracts?

Reply:
Each network has a configuration (ZC contracts, tokens address, and others). In this case, the configuration is for Chainlink, but it is not available in Ganache (also for example the token DAI).

That configuration is used to run Truffle scripts that interact with already deployed contracts.

I will create a dummy address (const) in a consts.js file. So, it can be imported in the conf files (and update the conf value when the contract is deployed).